src/controller/index.ts

```diff
@@ -214,6 +218,27 @@ export default class SimulariumController {
         }
     }
 
+    public convertAndLoadTrajectory(
+        netConnectionConfig: NetConnectionParams,
+        obj: Record<string, unknown>,
+        fileType: string
```

Comment:
we mentioned that it would be good to have this typed as an enum that currently has just one item, but to make it clear that fileType has to be one of our specified engines